doc/source/examples/pdp_regression_bike.ipynb

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
<p>I would suggest computation and visualization to be in separate cells as best practice, especially in cases where computation can take some time and here the user may want to tweak the figure parameters without recomputing.</p><p> </p><p>Also noting that the plot for season does not start at 0 unlike the first 3 plots which can be misleading (looks like a huge drop).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>yes, it doesn't start at 0 because the <code>sharey="row"</code>. I could set it to <code>"all"</code> but then you would not see any drop because it is quite small. For me, as long the y axis shows the values I can understand it easily, but if you think that this is still confusing after clarifying why the second row doesn't start at 0, I will change it to <code>"all"</code>.      </p><p>  </p>